pkg/env/process_pruning.go

```diff
@@ -1,8 +1,35 @@
 package env
 
+import "time"
+
 var (
 	// ProcessPruningEnabled toggles whether process pruning should be done periodically using heuristics
 	// This may be useful in certain large environments, but is fairly expensive as it requires a full
 	// sweep over the database
 	ProcessPruningEnabled = RegisterBooleanSetting("ROX_PROCESS_PRUNING", false)
+
+	// How often to perform pruning
+	PruneInterval = registerDurationSetting("ROX_PRUNE_INTERVAL", 1*time.Hour)
+
+	// Timeout for reading part of pruning query
+	PruneOrphanedTimeout = registerDurationSetting("ROX_PRUNE_ORPHANED_TIMEOUT", 5*time.Minute)
```

Comment:
Probably useful clearly differentiate this from the orphaned window.
```suggestion
	PruneOrphanedQueryTimeout = registerDurationSetting("ROX_PRUNE_ORPHANED_QUERY_TIMEOUT", 5*time.Minute)
```